tools/defaults.bzl

```diff
@@ -245,9 +245,7 @@ def ng_web_test_suite(deps = [], static_css = [], bootstrap = [], **kwargs):
 
     karma_web_test_suite(
         # Depend on our custom test initialization script. This needs to be the first dependency.
-        deps = [
-            "//test:angular_test_init",
-        ] + deps,
+        deps = deps if exclude_init_script else ["//test:angular_test_init"] + deps,
```

Comment:
This change is necessary so that the test setup is identical between the local environment and Stackblitz.